### PR TITLE
DEV-34633 Downgrade spotless to be Java 8 compatible

### PR DESCRIPTION
**Pull request by willynganga**

### CODE REVIEW

**Review by Thilo-Acrolinx — APPROVED** [review]

File: .github/workflows/gradle.yml [hunk 1]
```diff
@@ -10,27 +10,48 @@ on:
     - cron: '0 3 * * *'
 
 jobs:
+  Sonar-Check:
+    runs-on: ubuntu-latest
+    env:
+      SONAR_TOKEN: ${{ secrets.SONARCLOUD_TOKEN }}
```
Comment (Thilo-Acrolinx) [inline .github/workflows/gradle.yml:16]: we have this here and again on line 33 below?

File: .github/workflows/gradle.yml [hunk 2]
```diff
@@ -10,27 +10,46 @@ on:
...
+
   build:
     runs-on: ubuntu-latest
 
     steps:
       - uses: actions/checkout@v3
         with:
           fetch-depth: 0
+
       - name: Set up JDK 11
```
Comment (Thilo-Acrolinx) [inline .github/workflows/gradle.yml:43]: omit the "11"

Comment (willynganga) [reply .github/workflows/gradle.yml]: Is setup JDK 8 okay?

Comment (Thilo-Acrolinx) [reply .github/workflows/gradle.yml]: At least it is correct and not wrong, like 11 was. :-) I am OK to leave it like this.

Maybe just saying "Set up JDK" would be enough, if one needs to know the version it is best to check the yml file for the truth. And then next time we change the JDK version we avoid the risk of having an outdated name again.

**Review by Thilo-Acrolinx — APPROVED** [review]

File: .github/workflows/gradle.yml (hunk 2 above)
Comment (willynganga) [reply .github/workflows/gradle.yml]: That makes sense. I think when updating the Java version, it would be best to also update the stage name.

**Pull request merged**